Code/LauncherUnified/launcher_generator.cmake

```
@@ -109,16 +125,33 @@ foreach(project_name project_path IN ZIP_LISTS O3DE_PROJECTS_NAME LY_PROJECTS)
             Legacy::CrySystem
         )
 
+        # in script only mode, which can only happen when building a project
+        # that is using a prebuilt installer, add the generic gamelauncher as a run time dependency
+        # of the project launcher.  The project launcher is "fake" and will not get actually compiled,
+        # but this will cause the generic game launcher and its dependencies to get deployed into the bin folder
+        # since they are all dependencies of this fake target.
+        
+        if (O3DE_SCRIPT_ONLY)
+            set(game_runtime_dependencies ${game_runtime_dependencies} O3DE.GameLauncher)
+        endif()
+
         if(PAL_TRAIT_BUILD_SERVER_SUPPORTED)
             set(server_runtime_dependencies
                 Legacy::CrySystem
             )
+
+            if (O3DE_SCRIPT_ONLY)
+                set(server_runtime_dependencies ${server_runtime_dependencies} O3DE.ServerLauncher)
+            endif()
```

This needs to use `SERVER_LAUNCHER_TYPES` since there are more than one server launcher type.
```suggestion
            if (O3DE_SCRIPT_ONLY)
                foreach(server_launcher_type ${SERVER_LAUNCHER_TYPES})
                    set(server_runtime_dependencies ${server_runtime_dependencies} O3DE.${server_launcher_type})
                endforeach()
            endif()
```

---

I'm actually not sure this is 100% correct, I'm changing it to 
```cmake
if (O3DE_SCRIPT_ONLY)
                foreach(server_launcher_type ${SERVER_LAUNCHER_TYPES})
                    set(SERVER_RUNTIME_DEPENDENCIES_${server_launcher_type} "${SERVER_RUNTIME_DEPENDENCIES_${server_launcher_type}}" O3DE.${server_launcher_type})
                endforeach()
            endif()
            ```
            
 and then later on, changing it to use
 ```
 RUNTIME_DEPENDENCIES
                    ${SERVER_RUNTIME_DEPENDENCIES_${server_launcher_type}}
                    ```